app/controllers/admin/organizations_controller.rb

```diff
@@ -28,6 +28,10 @@ def show
     @org = Organization.find(params[:id])
     authorize @org
     prep_form_vars
+
+    @notes = @org.notes.order(created_at: :desc)
```

Comment:
Have you checked for N+1 here? I bet you need to load at least the author and notable of each note due to the policy check. Or maybe your fancy loader thing is handling it ;) Worth looking at the log.